api/ts/src/gen/pub/sandbox/v1alpha1/sandbox_pb.ts

```diff
@@ -320,6 +336,14 @@ export class ListSandboxesRequest extends Message<ListSandboxesRequest> {
    */
   fetchStatusAndUrl = false;
 
+  /**
+   * Optional. If passed in, only sandboxes for this project are returned.
+   * Project must belong to organization
+   *
+   * @generated from field: string project_id = 2;
+   */
+  projectId = "";
```

Comment:
What about createSandbox, can an optional projectId be passed in?

Reply:
yep, both `name` and `projectId` are added to create action.